_posts/2020-11-07-brief-history-of-programming-languages.md

```diff
@@ -0,0 +1,103 @@
+
+Late 1960s to late 1970s flourished with programming languages, many of which are still used today.
+
+## BASIC
+
+BASIC, which stands for Beginner’s All-purpose Symbolic Instruction Code, was developed in 1964 by John G. Kemeny and Thomas E. Kurtz for students to help with understanding computers, even without advanced skillset in mathematics. Bill Gates and Paul Allen modified BASIC and became the first Microsoft product.
+
+## Pascal
+
+Pascal was named after the French mathematician, physicist and philosopher Blaise Pascal. The language was developed in 1970 by Niklaus Wirth and created as a tool for teaching computer programming. In Apple's early years it was used as a main language for their software programming.
+
+## C
+
+C was developed in 1972 by Dennis Ritchie at Bell Labs in order to construct utilities running on Unix. This high-level language is closer to human language and it influenced a variety of other languages such as c#, Go, Java, Ruby, Perl, Python and JavaScript.
+
+## SQL
+
+SQL was developed in 1972 by Donald D. Chamberlin and Raymond F. Boyce at IBM. SQL stands for Structured Query Language and it is used for viewing, changing and managing data stored in databases.
+
+## MatLab
+
+MATLAB stands for Matrix Laboratory and it was developed by Cleve Moler in 1978. It is widely used for mathematics, research and education as it is capable of creating two- and three-dimentional graphics. It allows for matrix manipulations, implementing of algorithms, plotting data and interfacing with programs written in other languages.
+
+## Objective-C
+
+Objective-C was developed in 1983 by Brad Cox and Tom Love and it is a general-purpose programming language that adds Smalltalk-style messaging to the language. It is the main language supported by Apple for macOS and iOs, the operating system of Apple.
+
+## C++
+
+Developed by Bjarne Stroustrup in 1983 and based on C language, C++ is one of the most widely used languages in the world. Game engines and high-performance software such as Adobe Photoshop are in C++.
+
+## Perl
+
+Perl was developed by Larry Wall in 1987 as a scripting language to make report processing easier. Nowadays it is used for Linux system administration, Web development  and network programming. Perl is actually a family of languages that include features from other languages including C, shell script, AWK and sed. Perl 5 became popular in late 1990s as a CGI scripting language, due to its regular expression and string parsing abilities.
```

Comment:
Extra space after "development" in "Web development  and network programming"

Reply:
Thanks!